easse/cli.py

```diff
@@ -3,19 +3,24 @@
 
 from easse.annotation.word_level import corpus_analyse_operations
 from easse.fkgl import corpus_fkgl
-from easse.utils.helpers import read_lines
+from easse.utils.helpers import read_lines, read_split_lines, collapse_split_sentences
 from easse.quality_estimation import corpus_quality_estimation
 from easse.sari import corpus_sari
 from easse.samsa import corpus_samsa
-from easse.utils.constants import VALID_TEST_SETS, VALID_METRICS, DEFAULT_METRICS, TEST_SETS_PATHS
+from easse.splitting import corpus_macro_avg_sent_bleu, sys_length_statistics, ref_length_statistics
+from easse.compression import corpus_macro_avg_f1_token, corpus_macro_avg_compression_ratio
+from easse.utils.constants import VALID_TEST_SETS, VALID_METRICS, DEFAULT_METRICS
 from easse.utils.resources import get_orig_sents, get_refs_sents
 from easse.report import write_html_report
 
 
 def get_sents(test_set, orig_sents_path=None, sys_sents_path=None, refs_sents_paths=None):
     # Get system sentences to be evaluated
     if sys_sents_path is not None:
-        sys_sents = read_lines(sys_sents_path)
+        if test_set in ['wikisplit', 'wikisplit_valid']:
```

Comment:
The WikiSplit dataset has `split_sep='<::::>'` in all its sentences. This helps, in particular, for calculating the `length_stats` metrics, since there is no need to split the sentences with any external function (which could introduce errors). However, besides those metrics, the other ones are ngram-based (like BLEU or sentBLEU) and we only need the `collapsed` versions of the sentences. 
If we stored WikiSplit already collapsed, we would need to: (1) rely on an external function to split the sentences for the cases of `length_stats` which could introduce errors; or (2) somehow read a collapsed and a non-collapsed version of the dataset, which would require modifying the general function that reads all datasets (and add some special cases which we are trying to avoid).
Another option could be to have a map dataset-read_function, that stores which function to call when reading a specific dataset. This could add some flexibility for future datasets, too. This could avoid some `if`s, but perhaps not the one that determines whether the sentences needs to be collapsed or not. Any suggestions?